.circleci/config.yml

```diff
@@ -27,9 +27,21 @@ jobs:
       - run:
           name: Build base image
           command: docker build --build-arg GRAPHHOPPER_TOKEN=$GRAPHHOPPER_TOKEN --build-arg GIT_USER_ID=$GIT_USER_ID --build-arg GIT_PERSONAL_ACCESS=$GIT_PERSONAL_ACCESS . -f Dockerfile.build -t us.gcr.io/model-159019/gh:$(cat /tmp/tag)_build
+      - run: # must install sudo first; see https://discuss.circleci.com/t/sudo-command-not-found/14208/4
+          name: Download + setup micro norcal test files
+          command: |
+            apt-get install -y sudo
+            ls -al /bin/sh && sudo rm /bin/sh && sudo ln -s /bin/bash /bin/sh && ls -al /bin/sh
+            ./setup_tests.sh
       - run:
           name: Run Tests
-          command: docker run us.gcr.io/model-159019/gh:$(cat /tmp/tag)_build /bin/bash -c "mvn test"
+          command: |
+            docker run -t -d --name testrunner us.gcr.io/model-159019/gh:$(cat /tmp/tag)_build /bin/bash
```

Comment:
ahh i see what's happening now. some of the docker commands are a little funky. This is mostly to avoid keeping the test data in the repo? (How big is it?) I'm happy if this works, but out of curiosity is there any way to encapsulate this in a test dockerfile that used the base image?

Reply:
See my other comment for more reasoning to why I took this approach.

Hmmmm, now that you mention it, though....micro_nor_cal could probably fit in GH directly 😳 . OSM is only ~40mb...so, mini_nor_cal wasn't possible, but micro probably is. Perhaps that changes the approach here, I guess we could just generate it manually with the script and plop it in the repo....

re `is there any way to encapsulate this in a test dockerfile that used the base image?`, what do you mean exactly? An image that contains the test fixture data + has access to `mvn` to run tests against? That's essentially what we're building with the commands I have in this section - a few changes and we could push the result, then run `mvn test` against it (instead, we're just "building" it live in a container and then running the test command)

Reply:
I think it's fine to move forward with this. we can reorganize in the future if necessary. I was trying to think of a better way to encapsulate the ci commands in a new test dockerfile to streamline the circleconfig.